### PR TITLE
feat(conf): resolve labels values with env vars

### PR DESCRIPTION
**Pull request by miton18**
Support env vars replacement in labels values

Ex:
```yaml
labels:
    another: env:USER
```

### CODE REVIEW

**Comment by PierreZ** [comment]
I'm wondering if we should be prefixing all env by `BEAMIUM_` and enable it through a flag like `forward_env: true` for a scraper. This is be easier IMO than generating a whitelist through configuration with the `env:` prefix.

**Comment by miton18** [comment]
yes it easier to use, but with it, how can you choose between a scrapper specific label and a global one ?

**Comment by PierreZ** [comment]
You are right. I just don't find it useable to specify env to select per scraper through configuration. Maybe something like:

* BEAMIUM_LABELS_GLOBALS_TOTO
* BEAMIUM_LABELS_SCRAPER_NAME_TOTO

?

**Comment by miton18** [comment]
@PierreZ both syntax are now supported, you can either use`env:toto` in the config file
or add en en var with the prefix `BEAMIUM_LABEL_XXXXX`

File: README.md [hunk 1]
```diff
@@ -66,6 +66,14 @@ In addition, it will recursively discover configuration files in `beamium.d` dir
 
 Furthermore, Beamium support multiple formats for configuration files which are `hjson`, `json`, `toml`, `yaml`, `yml` or `ini`.
 
+
+Also, Beamium can be started with several labels put as env vars, they must be prefixed by `BEAMIUM_LABEL`.
+
+Ex:
+```sh
+BEAMIUM_LABEL_HOST=myhost ./beamium -v
```
Comment (FlorentinDUBOIS) [inline README.md]: You spoke about some others behaviours, could you write it there too?

File: src/conf.rs [hunk 2]
```diff
@@ -553,3 +566,37 @@ impl Conf {
...
+    if !value.starts_with("env:") {
+        return value;
+    }
+
+    let striped = value.trim_start_matches("env:");
+
+    return match env::var(striped) {
+        Ok(v) => v,
+        Err(_e) => {
+            striped.to_owned().push_str("_NOT_SET");
```
Comment (FlorentinDUBOIS) [inline src/conf.rs]: I prefer to drop the entry and use `warn!("env variable in not set ...")`

File: README.md (hunk 1 above)
Comment (miton18) [reply README.md]: the 2nd one is described here https://github.com/ovh/beamium/pull/119/files#diff-04c6e90faac2675aa89e2176d2eec7d8R90

**Pull request merged**